code/API_definitions/check-device-connectivity.yaml

```diff
@@ -115,6 +155,18 @@ components:
           $ref: '#/components/schemas/CountryCode'
         countryName:
           $ref: '#/components/schemas/CountryName'
+    NetworkConnectionEventReport:
+      type: "object"
+      properties:
+        ueId:
+          $ref: "#/components/schemas/UeId"
+        uePort:
+          $ref: '#/components/schemas/Port'
+        eventType:
+          $ref: "#/components/schemas/NetworkConnectionEventType"
+      required:
+        - ueId
+        - eventType      
```

Comment:
This response just contains one device. 
But it's required to have a list of devices containing only the properties ```ueId``` and ```connected``` as the boolean.